### ISSUE TITLE
"#!/bin/bash" should be excluded by default

### ISSUE THREAD

**Issue by 7Pawns** [aaron-bond/better-comments#514]
![image](https://github.com/aaron-bond/better-comments/assets/124451776/d4407fd3-2085-4bc8-9b3c-9d6c3524bd09)

Doesn't make much sense to keep it like that by default

**Comment by ppo** [comment]
Worse!… `#!/…` first thing in the file (first character on first line) should be excluded.

**Comment by tylerlaprade** [comment]
I don't think this repo is maintained anymore. I've switched over to https://github.com/edwinhuish/better-comments-next instead.

**Comment by ppo** [comment]
Thanks @tylerlaprade! Way better… I saw the "next" version when installing that extension but too lazy “I'll check that later”… then I forgot 🙄😂
